kmk/firmware.py

```diff
@@ -95,10 +120,10 @@ def _send_key(self, key):
             self._send_hid()
 
     def _handle_matrix_report(self, update=None):
-        '''
+        """
```

Comment:
There are basically zero instances I like to see double quotes in this project - if this was done through an automated tool, let me know so I can add a config file to kill that behavior.

Single quotes are standard across almost all core code (at least... this _was_ the case) and should remain that way.

Reply:
Flake complained. I guess I should have told that to ignore.

Reply:
Fixed. No double quotes exist except your personal files, to which I don't care.